docs/src/content/docs/api/tools.mdx

```diff
@@ -210,6 +213,92 @@ You then pass in the key(s) in the `authTokens` property of `selectedTools` when
   </TabItem>
 </Tabs>
 
+## Tool Parameters
+Tool parameters define what gets passed in to your backend service when the tool is called. When creating a tool, parameters are defined as one of three types:
+<Steps>
+1. **Dynamic** → The model will choose which values to pass. Can be overridden (see [below](./#parameter-overrides)).
+1. **Static** → Value is known and unconditionally set when the call starts. Not exposed to or set by the model.
+1. **Automatic** → Automatically set by the system. Not known when the call starts but known at tool invocation.
+</Steps>
+
+### Dynamic Parameters
+Dynamic parameters will have their values set by the model. Creating a dynamic parameter on a tool looks like this:
+
+```javascript
+// Adding a dynamic parameter to a stock price tool
+{
+  "name": "stock_price",
+  "description": "Get the current stock price for a given symbol",
+  "dynamicParameters": [
+    {
+      "name": "symbol",
+      "location": "PARAMETER_LOCATION_QUERY",
+      "schema": {
+        "type": "string",
+        "description": "Stock symbol (e.g., AAPL for Apple Inc.)"
+      },
+      "required": true
+    }
+  ]
+}
+```
+
+#### Parameter Overrides
+You can choose to set static values for dynamic parameters when you start a call. The model won't see any parameters that you override. When creating a call simply pass in the overrides with each tool:
+
+```javascript
+// Overriding dynamic parameter when starting a new call
+// Always set the color to 'green'
+{
+  "model": "fixie-ai/ultravox-70B",
+  "systemPrompt": ...
+  "selectedTools": [
+    "toolName": "colorPicker",
+    "parameterOverrides": {
+      "color": "green"
+    }
+  ]
+}
+```
+
+Parameter overrides don't make sense for temporary tools. Instead of overriding a dynamic parameter, use a static parameter instead.
+
+### Static Parameters
+If you have parameters that are known at the time you create the tool, static parameters can be used.
+
+```javascript
+// Adding a static parameter to a stock price tool
+{
+  "name": "stock_price",
+  "description": "Get the current stock price for a given symbol",
+  "staticParameters": [
+    {
+      "name": "close_date",
```

Comment:
This seems like something that ought to be an automatic parameter (not that it's something we support right now - just seems like we could have a better example).  Should this maybe be start_date?  Or maybe something more generic like adding a utm=ultravox query param?

Reply:
yeah will change to utm idea